README.md

```diff
@@ -657,12 +665,108 @@ Basic usage: `amclient.py <subcommand> [optional arguments]
     * `--ss-url <url>` - Storage Service URL (default: `http://127.0.0.1:8000`)
     * `--directory <dir>` - directory path in which to save the DIP
 
+* get-pipelines
+  * purpose: list (enabled) Pipelines known to the Storage Service.
+  * positional arguments:
+      * ss_api_key - Storage Service API key
+  * optional arguments:
+    * `--ss-user-name <username>` - Storage Service username (default: `test`)
+    * `--ss-url <url>` - Storage Service URL (default: `http://127.0.0.1:8000`)
+
+* get-transfer-status
+  * purpose: print the status of a transfer if it exists in a transfer
+  workflow.
+  * positional arguments:
+      * transfer_uuid - identifier for the transfer in progress
+      * am_api_key - API key for Archivematica dashboard user
+  * optional arguments:
+    * `--am-user-name <username>` - username for Archivematica dashboard user
+    * `--am-url <url>` - Archivematica URL (default: `http://127.0.0.1`)
+
+* get-ingest-status
+  * purpose: print the status of an ingest if it exists in an ingest workflow.
+  * positional arguments
+    * sip-uuid - identifier for the ingest in progress
+    * am_api_key - API key for Archivematica dashboard user
+  * optional arguments
+    * `--am-user-name <username>` - username for Archivematica dashboard user
+    * `--am-url <url>` - Archivematica URL (default: `http://127.0.0.1`)
+
+* get-processing-config
+  * purpose: print a processing configuration file given its name in
+  Archivematica.
+  * positional arguments
+    * am_api_key - API key for Archivematica dashboard user
+  * optional arguments
+    * processing-config
+    * `--am-user-name <username>` - username for Archivematica dashboard user
+    * `--am-url <url>` - Archivematica URL (default: `http://127.0.0.1`)
```

Comment:
Again, not your responsibility here and now, but seeing all this repetition makes me think a) that a lot of it could be removed and b) that we should allow for the specification of a config file so that all but the essential becomes optional.

Reply:
Logged in #84